defaults/main.yml

```diff
@@ -1,2 +1,6 @@
 # defaults file
 ---
+
+aws_efs_utils_version: v1.31.3
+efs_mount_dir: /efsmnt
+efs_file_system_id:
```

Comment:
perhaps rename this to `efs_src` since we can mount not only via fs id?

Reply:
efs_file_system_id is prefered one and it will not be confusing. We can add another variable but then we need to create more complex mount command. But this is a good point. We can create issue add this in the next PR e.g. wdyt?